bigtop-deploy/vm/vagrant-puppet-vm/Vagrantfile

```diff
@@ -112,7 +113,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
       bigtop.vm.provider :virtualbox do |vb|
         vb.customize ["modifyvm", :id, "--memory", CONF['memory_size']]
-	vb.customize ['modifyvm', :id, '--cpus', CONF['number_cpus']]
+        vb.customize ['modifyvm', :id, '--cpus', CONF['number_cpus']]
```

Comment:
Sure, I will undo the change. Its not an accidental change. The file is using spaces for indentation, and only this line uses tabs (which makes it look unaligned in some editors)